### PR TITLE
Make it clear that the mute is automatic

### PR DESCRIPTION
**Pull request by torchgm**
Some peeps have occasionally confused the no-DM automute message for another manual mute, might as well make it clearer that it isn't.

### CODE REVIEW

File: Modules/Mutes.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public static async System.Threading.Tasks.Task<bool> MuteUserAsync(DiscordMembe
             {
                 // A DM failing to send isn't important, but let's put it in chat just so it's somewhere.
                 if (!(channel is null))
-                    await channel.SendMessageAsync($"{Program.cfgjson.Emoji.Muted} {naughtyMember.Mention} was muted for **{Warnings.TimeToPrettyFormat(muteDuration, false)}**!");
+                    await channel.SendMessageAsync($"{Program.cfgjson.Emoji.Muted} {naughtyMember.Mention} was automuted for **{Warnings.TimeToPrettyFormat(muteDuration, false)}**!");
```
Comment (Erisa) [inline Modules/Mutes.cs:75]: I'd prefer something less slang like "automatically muted".
Maybe could also show that it was for multiple warns by borrowing the text from the DM?

**Review by Erisa — CHANGES_REQUESTED** [review]

File: Modules/Mutes.cs (hunk 1 above)
Comment (Erisa) [reply Modules/Mutes.cs:75]: Also I'm pretty sure modifying this text will affect all mutes? Not just automatic ones

**Comment by Erisa** [comment]
I'll check this later on PC, thanks

File: Modules/Mutes.cs (hunk 1 above)
Comment (torchgm) [reply Modules/Mutes.cs]: > Also I'm pretty sure modifying this text will affect all mutes? Not just automatic ones

I thought so too but after initiating a manual mute it seems to print the string "has been muted" as opposed to "was muted" which seems to happen [here](https://github.com/Erisa/Cliptok/blob/main/Modules/Mutes.cs#L246).
image

Comment (Erisa) [reply Modules/Mutes.cs]: This made me think though - what happens if you manually mute someone who's blocking DMs?

Comment (torchgm) [reply Modules/Mutes.cs]: Absolutely no idea, I guess it prints two messages?

Comment (Erisa) [reply Modules/Mutes.cs]: That was my thought and was why I asked :P either way I'll look at it later

**Comment by Erisa** [comment]
Tbh I kinda hijacked your PR to do it a more complicated way while also fixing a related bug.

But hey, the title of the PR is implemented so I call this a win.

**Pull request merged**